### ISSUE TITLE
git pull --rebase

### ISSUE THREAD

**Issue by mrozsz** [zaggino/brackets-git#938]
I call *git pull --rebase <origin> <branch>* and if there are any changes I get the error.

My local branch is different from remote.
local: *my-test* 
remote: *feature/test-protototype*

You can review it, if you want.

Thanks, Mrozsz

--------------------------------

*Brackets 1.1.0-15558 (win), Git 1.9.5.msysgit.0 & Brackets-Git 0.14.17*

```
Error: Failed parsing branch name from 7526cc025228e9f87bf06870af21be9577d0ca7f
```

```
Error: Failed parsing branch name from 7526cc025228e9f87bf06870af21be9577d0ca7f
    at file:///C:/Users/ofrinta/AppData/Roaming/Brackets/extensions/user/zaggino.brackets-git/src/Branch.js:386:40
    at File.eval (/filesystem/File.js:125:13)
    at eval (/filesystem/impls/appshell/AppshellFileSystem.js:378:25)
```

**Comment by zaggino** [comment]
Note to self: this happens because of the watcher on the branch name

**Comment by sabrehagen** [comment]
+1 get this any time i try to rebase

**Comment by zaggino** [comment]
This should be fixed in latest version `0.14.19`, please test/confirm. Thanks!

**Comment by mrozsz** [comment]
I didn't see the error during latest rebases, so I think that it is ok.Thanks.
